### ISSUE TITLE
fish shell command completion

### ISSUE THREAD

**Issue by jwulf** [serverless/serverless#248]
[Fish shell] (http://fishshell.com) is the bomb! I cannot believe I have lived this long without it. 

`omf install jaws-cli` 

for command completion!

**Comment by theopak** [comment]
Does that command work for you? It doesn't work for me, and I can't find "jaws-cli" in the global registry. You could submit a PR on oh-my fish :smile: https://github.com/oh-my-fish/oh-my-fish/blob/master/docs/en-US/Packages.md#make-it-public

```bash
$ omf install jaws-cli
Installing package jaws-cli
fatal: repository 'jaws-cli' does not exist
Could not install package jaws-cli.
```
